openslam_gmapping/gridfastslam/gfsreader.cpp

```diff
@@ -501,6 +501,6 @@ void RecordList::printPath(ostream& os, unsigned int i, bool err, bool rawodom)
 		cout << "average error" << totalError/count << endl;
 }
 
-}; //gfsreader
```

Comment:
Since openslam_gmapping has been directly adopted, we can keep the original formatting of the code. This also applies for all the other openslam_gmapping files that have been changed

Reply:
I agree. I have reverted this to adopt the original code.